src/iv/imageviewer.cpp

```diff
@@ -93,7 +95,13 @@ static const char *s_file_filters = ""
 
 
 
-ImageViewer::ImageViewer()
+ImageViewer::ImageViewer(
+#ifdef USE_OCIO
+    bool use_ocio, const std::string& image_color_space,
+    const std::string& display, const std::string& view
+/*, const std::string& look */
+#endif
+    )
```

Comment:
To keep the `#if` pollution to a minimum and improve readability, I think it's probably fine here to just keep all the parameters and data members unconditionally, but only *use* them if OCIO support is built in and of the sufficient version.

Reply:
Will do!